packages/nx-aws-cache/src/tasks-runner/runner.ts

```diff
@@ -1,5 +1,12 @@
+import { existsSync } from 'fs';
 import { config as dotEnvConfig } from 'dotenv';
-dotEnvConfig();
+
+const envPaths = ['.local.env', '.env.local', '.env'];
+const envConfig = envPaths.find((path) => existsSync(path));
```

Comment:
This loads only one of the configs right? We still should load all of them, only in the right order, as if `.env.local` exists, still `.env` variables should be loaded as well. One example is here, but I'm not sure if the order of them is correct: https://github.com/nrwl/nx/blob/fa513731d4e6a960f112a5af6b0d725b2e624b36/packages/nx/bin/nx.ts#L110-L116

Reply:
@ddivecs any update on this?

Reply:
You're correct, ill fix this and update soon. Sorry for my slow responses.

Reply:
it seems the documentation doesn't match what nx actually wrote as code, the docs https://nx.dev/recipes/tips-n-tricks/define-environment-variables list .local.env > .env.local > .env but the code does .env.local > .local.env > .env.

This seems like an issue thats already been filed as well. https://github.com/nrwl/nx/issues/18707
Im going to follow the documentation, not the actual code, but we should watch that issue and see what changes come from it and correct as necessary.